README.md

```diff
@@ -25,7 +25,7 @@ __Notes:__ Issues with the output should be reported on the babel [issue tracker
 > webpack 3.x | babel-loader 8.x | babel 7.x
```

Comment:
Nice. You might want to add 4.x to the version list - seems to be working fine for me, at least.